kairos-session-code/rust-toolchain.toml

```
@@ -0,0 +1,4 @@
+[toolchain]
+channel = "nightly-2024-04-09"
```

There is no need to use `nigthly` for our session codes.

```suggestion
channel = "stable"
```

---

Fixed in https://github.com/cspr-rad/kairos/pull/120/commits/ad2802ccf96a86b23fd07c112a823220e1de7d54